packages/actor-query-operation-group/lib/ActorQueryOperationGroup.ts

```diff
@@ -43,7 +43,8 @@ export class ActorQueryOperationGroup extends ActorQueryOperationTypedMediated<A
       ...aggregates.map(agg => termToString(agg.variable)),
     ];
 
-    const sparqleeConfig = { ...ActorQueryOperation.getExpressionContext(context) };
+    const sparqleeConfig = { ...ActorQueryOperation.getExpressionContext(context),
+      extensionFunctionCreator: undefined };
```

Comment:
Why are you manually overriding extensionFunctionCreator here?

Reply:
Currently the `ActorQueryOperationGroup` uses a sync `AggragateEvaluator` in sparqlee. We can not provide the async extension functions to this aggragator. I currently override extensionFunctionCreator here because it's the wrong type. Sparqlee should in fact also have an AsyncAggragateEvaluator so we can use our async extensionFunctionCreator without problems.

Reply:
Check.
Could you mark this with a `TODO`?

Reply:
Sure!